src/CMDParse.cpp

```diff
@@ -0,0 +1,57 @@
+#include "CMDParse.h"
+
+EERAModel::ArgumentParser::ArgumentParser(int argc, char** argv)
+{
+    try
+	{
+		TCLAP::CmdLine cmd("A model structure argument and mode of running must be provided",   // Help string message
```

Comment:
This doesn't actually match the message on line 20, which implies that it's okay to leave out the mode argument.